swagger.yaml

```diff
@@ -146,7 +145,19 @@ components:
         format_version:
           type: string
           enum: ["1.0.0"]
+        rule_bundles:
+          type: array
```

Comment:
In this case I'm sort of leaning towards making this an object with an "opaque" string key, and then having the rules just include this key.  I'd also like to fold the errors in here as an additional field.  Maybe there's a reason that doesn't work out though?

Reply:
The reasons that I shaped things this way are:
* I'm still not sure what fields we'll want to use to uniquely identify bundles, so I opted to just use the fields that I think are components of that identity.
* It's possible for a bundle to be specified multiple times and since our rule results are also an array, you would see duplicate rule results. Also having the bundle show up multiple times makes it clearer why that happened.

I am open to using a string to relate rules back to bundles as long as we make it clear that it's not meant to be interpreted as a stable identifier for the bundle.

Reply:
Hmm, I see your point and I'm also fine using an array for this, but then I would lean towards using a different type at the top level (that can contain more info, including errors) than the one included in the rule result (`RuleResultBundleInfo`)?

Reply:
That makes sense! I've tried to consolidate these in bc8bccc.